rules/windows/process_creation/win_susp_powershell_enc_cmd.yml

```diff
@@ -16,27 +16,27 @@ logsource:
     product: windows
 detection:
     selection:
-        CommandLine:
-            - '* -e JAB*'
-            - '* -e  JAB*'
-            - '* -e   JAB*'
-            - '* -e    JAB*'
-            - '* -e     JAB*'
-            - '* -e      JAB*'
-            - '* -en JAB*'
-            - '* -enc JAB*'
-            - '* -enc* JAB*'
-            - '* -w hidden -e* JAB*'
-            - '* BA^J e-'
-            - '* -e SUVYI*'
-            - '* -e aWV4I*'
-            - '* -e SQBFAFgA*'
-            - '* -e aQBlAHgA*'
-            - '* -enc SUVYI*'
-            - '* -enc aWV4I*'
-            - '* -enc SQBFAFgA*'
-            - '* -enc aQBlAHgA*'
+        CommandLine|contains:
+            - ' -e'
+            - ' -en'
+            - ' -enc'
+            - ' -w hidden -e'
+    selection2:
+            - 'JAB'
+    selection3:
+            - '-e'
```

Comment:
`-e` covers `-enc`
the same for `selection` (without number)

Reply:
Fixed